hyperspy/_signals/eds_tem.py

```diff
@@ -886,21 +889,20 @@ def CL_get_mass_thickness(self, weight_percent, thickness):
 
```

Comment:
the two comment lines above this line (Input given by ... ... unit electric charge) are not relevant for this function.

Reply:
Done!